HierarchicalClustering/tests/test_random_walks.py

```diff
@@ -1,127 +1,131 @@
+import unittest
+
 import pytest
 import random
 import numpy as np
 import math
 
-from HierarchicalClustering.GraphObjects import EnhancedHypergraph
-from HierarchicalClustering.js_divergence_utils import _kl_divergence, _js_divergence
+from GraphObjects import Hypergraph
+from js_divergence_utils import kl_divergence, js_divergence
+from Communities import Communities, run_random_walks, get_close_nodes, cluster_nodes_by_truncated_hitting_times
```

Comment:
I think you can just use from Communities import *

Reply:
Ok. Will hold off until we decide where these new functions live.